message_ix/util/__init__.py

```diff
@@ -46,28 +50,29 @@ def output(str):
     else:
         output = print
 
-    # Identify the source directory using importlib.resources
```

Comment:
Just recording for posterity that:
- `importlib.resources` was used in #784 because it's in the Python standard library, thus appears to be recommended/preferred over `pathlib.Path(__file__)`.
- However, there are [strong dissenting views](https://discuss.python.org/t/easy-and-recommended-way-to-get-path-of-datafile-within-package/20581/4) from Python users.
- It was decided to switch to using `pathlib.Path(__file__)`. This reverses changes made in the earlier PR.
- We don't know if this has any impact on flakiness of tests.